src/scripts/oauth_tables_script.sql

```diff
@@ -0,0 +1,97 @@
+
+DROP TABLE IF EXISTS app_clients CASCADE;
+
+CREATE TABLE app_clients(
+             id BIGSERIAL NOT NULL,
+             client_name VARCHAR(50) NOT NULL,
+             client_id TEXT  NOT NULL,
+             client_secret TEXT NOT NULL,
+             access_token_expiration_time BIGINT NOT NULL,
+             refresh_token_expiration_time BIGINT NOT NULL,
+             entity_status status_enum NOT NULL,
+             created_by BIGINT NOT NULL,
+             created_date TIMESTAMP WITHOUT TIME ZONE,
+             updated_by BIGINT NOT NULL,
+             updated_date TIMESTAMP WITHOUT TIME ZONE,
+             PRIMARY KEY (id)
+);
+
+DROP TABLE IF EXISTS app_clients_scopes CASCADE;
+
+CREATE TABLE app_clients_scopes(
+             id BIGSERIAL NOT NULL,
+             app_client_id BIGINT NOT NULL,
+             client_scope VARCHAR(50) NOT NULL,
```

Comment:
Rename to scope and add either a table of scopes or an enum

Reply:
done.